example-doc-win2019.md

```diff
@@ -0,0 +1,261 @@
+- Microsoft.VisualStudio.Component.VC.v141.x86.x64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ARM.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ATL.ARM.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ATL.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.ATL.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.MFC.ARM.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.MFC.ARM64.Spectre
+- Microsoft.VisualStudio.Component.VC.v141.MFC.Spectre
+- Microsoft.VisualStudio.Component.Windows10SDK.14393
+- Microsoft.VisualStudio.Component.Windows10SDK.16299
+- Microsoft.VisualStudio.Component.Windows10SDK.17134
+- Microsoft.VisualStudio.Component.Windows10SDK.17763
+- Microsoft.VisualStudio.Component.Windows10SDK.18362
+- Microsoft.VisualStudio.Component.WinXP
+- Microsoft.VisualStudio.ComponentGroup.Azure.CloudServices
+- Microsoft.VisualStudio.ComponentGroup.Azure.ResourceManager.Tools
+- Microsoft.VisualStudio.ComponentGroup.NativeDesktop.Llvm.Clang
+- Microsoft.VisualStudio.ComponentGroup.Web.CloudTools
+- Microsoft.VisualStudio.Workload.Azure
+- Microsoft.VisualStudio.Workload.Data
+- Microsoft.VisualStudio.Workload.DataScience
+- Microsoft.VisualStudio.Workload.ManagedDesktop
+- Microsoft.VisualStudio.Workload.ManagedGame
+- Microsoft.VisualStudio.Workload.NativeCrossPlat
+- Microsoft.VisualStudio.Workload.NativeDesktop
+- Microsoft.VisualStudio.Workload.NativeGame
+- Microsoft.VisualStudio.Workload.NativeMobile
+- Microsoft.VisualStudio.Workload.NetCoreTools
+- Microsoft.VisualStudio.Workload.NetCrossPlat
+- Microsoft.VisualStudio.Workload.NetWeb
+- Microsoft.VisualStudio.Workload.Node
+- Microsoft.VisualStudio.Workload.Office
+- Microsoft.VisualStudio.Workload.Python
+- Microsoft.VisualStudio.Workload.Universal
+- Microsoft.VisualStudio.Workload.VisualStudioExtension
+
+#### Microsoft SSDT Visual Studio 2019 Extensions
+- Microsoft Analysis Services Projects Version: 2.9.8
+- SQL Server Integration Services Projects Version: 3.6
+- Microsoft Reporting Services Projects Version: 2.6.5
+
+#### Windows Driver Kit
+- WDK Version: 10.1.18362.1
+- WDK Visual Studio Extension Version: 10.0.18346.0
+
+#### WIX Tools
+- Toolset Version: 3.11.4516
+- WIX Toolset Studio 2019 Extension Version: 1.0.0.4
+
+
+### .NET Core
+- SDK: 2.1.300, 2.1.301,... , 3.1.200, 3.1.201
+- SDK Path: C:\Program Files\dotnet\sdk
+- Runtime: 2.1.0, 2.1.1, 2.1.10, 2.111,... , 3.1.2, 3.1.3
+- Runtime Path: C:\Program Files\dotnet\shared\Microsoft.NETCore.App
+
+### .NET Framework
+- Version: 4.8.03761
+- Path: C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools
+
+### Package Management
+- Chocolatey 0.10.15
+- Vcpkg 2020.02.04
+- NPM 6.14.4
+- Yarn 1.22.4
+- Pip 20.0.2 (python 3.7.7)
+- Miniconda 4.6.14
+- RubyGems 3.1.2
+- Helm v3.2.0+ge11b7ce
+- Composer 1.10.5 2020-04-10 11:44:22
+
+### Project Management
+- Ant 1.10.5
+- Maven 3.6.3
+- Gradle 6.3
+
+### Tools
```

Comment:
Should we split this to Tools and Utilities like we do this in macOS docs?

Reply:
Agree, Tools section is pretty huge.
I would also add one more section: CLI tools

Reply:
Added `CLI tools` section. About `Utilities` section: it's unclear, how to separate software for `tools` and `utilities`